dockerfiles/che/Dockerfile.centos

```diff
@@ -46,7 +46,7 @@ EXPOSE 8000 8080
 
 USER user
 
-ADD assembly/assembly-main/target/eclipse-che-*/eclipse-che-* /home/user/che/
+ADD assembly /home/user/che
 
 ENTRYPOINT [ "/home/user/che/bin/che.sh", "-c" ]
```

Comment:
ok, I wanted to keep Dockerfile.centos as it was but if you say it's a mistake, then I will add usage of entrypoint.sh